src/StatusQ/Components/StatusExpandableSettingsItem.qml

```diff
@@ -0,0 +1,150 @@
+import QtQuick 2.14
+import StatusQ.Core 0.1
+import StatusQ.Controls 0.1
+import StatusQ.Core.Theme 0.1
+import StatusQ.Components 0.1
+
+Rectangle {
+    id: root
+
+    property alias icon: icon
+    property alias primaryText: primaryText
+    property alias secondaryText: secondaryText
```

Comment:
Unless there's a very good reason, we should not expose the entire component right away. So far it looks like the title and subtitle of this component always looks the same.

Can we start with `string` properties first and then go further if necessary?